### PR TITLE
Removing type from BaseFile and adding base_url variable

### PR DESCRIPTION
**Pull request by pradovc01**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -6,4 +6,5 @@ file.type-checksum=SHA-256
 file.tesseract-path=thirdParty/Tess4J/tessdata
 file.default-languageCode=eng
 file.default-language=english
+file.base-url=/api/v1.0/url
```
Comment (marylin-moya) [inline src/main/resources/application.properties]: why we have a property and constant with the same value? this property is never used

File: src/main/java/com/jalasoft/webservice/entitities/BaseFile.java [hunk 2]
```diff
@@ -18,20 +18,11 @@
  */
 public class BaseFile {
     private String fileName;
-    private String fileType;
     private String path;
     private String fullFileName;
```
Comment (marylin-moya) [inline src/main/java/com/jalasoft/webservice/entitities/BaseFile.java]: remove the fullFileName

File: src/main/resources/application.properties (hunk 1 above)
Comment (pradovc01) [reply src/main/resources/application.properties]: It was removed

File: src/main/java/com/jalasoft/webservice/entitities/BaseFile.java (hunk 2 above)
Comment (pradovc01) [reply src/main/java/com/jalasoft/webservice/entitities/BaseFile.java]: It was removed

**Pull request merged**